auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/model/SampleTypeWithParcelableContractSatisfied.java

```diff
@@ -0,0 +1,28 @@
+package com.ryanharter.auto.value.parcel.model;
```

Comment:
This should be able to be done via the standard testing methods, which aren't inner classes, but JavaObjects whose content is provided via String.